app/docs/enterprise/0.31-x/getting-started/adding-consumers.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Adding Consumers
+---
+
+# Adding Consumers
+
+<!-- <div class="alert alert-warning">
```

Comment:
we should remove this if we don't plan on showing it

Reply:
Well, we want to show it, but it need to link to a page that doesn't exist yet... shall I remove it until that other page exists?